packages/vite/src/node/config.ts

```diff
@@ -1063,9 +1064,18 @@ export async function loadConfigFromFile(
   }
 }
 
+async function getPathsFromClosestTsconfig(
+  resolvedPath: string,
+): Promise<string[]> {
+  const { tsconfig } = await parse(resolvedPath)
+  const paths = tsconfig?.compilerOptions?.paths || {}
```

Comment:
How about using the ?? operator instead of ||?
I think ?? would be more appropriate in this context.